test/functional/sqlsrv/TC79_RegressionVSTS846501.phpt

```diff
@@ -11,57 +11,50 @@ PHPT_EXEC=true
 <?php
 require_once('MsCommon.inc');
 
-function BugRepro()
+function bugRepro()
 {
-    include 'MsSetup.inc';
-
     $testName = "Regression VSTS 846501";
     startTest($testName);
 
     setup();
-    $conn1 = connect();
+    $conn1 = AE\connect();
     // empty parameter array
-    $s = sqlsrv_query($conn1, "DROP TABLE test_bq");
-    $s = sqlsrv_query($conn1, "CREATE TABLE test_bq (id INT IDENTITY NOT NULL, test_varchar_max varchar(max))");
+    $tableName = "test_bq";
+    dropTable($conn1, $tableName);
```

Comment:
I think createTable already drop the table before creating the table, so this line is not necessary.